Include/compile.h

```diff
@@ -23,8 +23,10 @@ extern "C" {
 #define PyCF_IGNORE_COOKIE 0x0800
 #define PyCF_TYPE_COMMENTS 0x1000
 #define PyCF_ALLOW_TOP_LEVEL_AWAIT 0x2000
+#define PyCF_DISABLE_ALL_OPTIMIZATIONS 0x3000
```

Comment:
It should be 0x4000.

Reply:
You may write a test similar to test_future.test_ensure_flags_dont_clash() to ensure that flags don't clash.